src/libultra/os/afterprenmi.c

```diff
@@ -1,5 +1,4 @@
-#include "ultra64.h"
-#include "global.h"
+#include "internal.h"
 
 s32 osAfterPreNMI(void) {
```

Comment:
So per the comment about includes https://github.com/zeldaret/oot/pull/1468#discussion_r1052701432
should this include `system.h` ? (with the `osAfterPreNMI` prototype)

The same kind of comment seems to apply to other files